src/WebUI/Resources.js

```diff
@@ -16,15 +16,14 @@ define("Environments/Providers/Kubernetes/Resources", ["require", "exports"], fu
     exports.ExternalIPText = "External IP";
     exports.TypeText = "Type";
     exports.AppNameText = "App Name";

```

Comment:
remove or change "App Name" to "AppName" or "App name"

Reply:
unused. removed.